modules/ROOT/pages/design-assigning-permissions.adoc

```diff
@@ -0,0 +1,49 @@
+= Set User Access Permissions
+
+Design Center provides multiple ways to set up access management to provide collaboration across teams. 
+
+Depending on your organization's permission structure, you can assign permissions using the xref:access-management::teams.adoc[Teams] feature, the xref:access-management::users.adoc#grant-user-permissions[Roles] feature (deprecated), or by assigning an xref:access-management::users.adoc#grant-user-permissions[individual user permission].
+
+NOTE: Mule Applications can't be shared.
+
+== Design Center Permissions
```

Comment:
We may need to add these in a subsection called business group permissions, and add another subsection for project level permissions. Also, we can clarify that whatever higher permission you are granted, that is the one that will be used. (i.e. User with DC Viewer that is project administrator of a certain project, will be able to administrate that project even if he/she is a DC Viewer in the org)